book_src/Operating System From 0 to 1.lyx

```diff
@@ -2643,6 +2644,10 @@ status collapsed
 \end_inset
 
 
+\begin_inset Separator latexpar

```

Comment:
Hmm, yeah. I'll try and copy the edited text and paste it in Vim. I'll also see if I can just download 2.1.